api/src/main/java/org/openmrs/module/ssemrreports/reporting/library/queries/CommonQueries.java

```diff
@@ -275,6 +275,14 @@ public static String getPatientsWithHighVL() {
 		return query;
 	}
 	
+	public static String getAllPatients() {
+		String query = "SELECT p.patient_id FROM patient p " + "LEFT JOIN encounter e ON p.patient_id = e.patient_id "
```

Comment:
Is there a reason why are you querying the transaction table instead of the etl table?

Also, perhaps you could rename this method to getPatientsWithEncountersBetweenDates.

Reply:
If the intention is to fetch all patients, the join with the encounter table is not necessary.

Reply:
> Is there a reason why are you querying the transaction table instead of the etl table?

Lemme query the ETL table instead

Reply:
> Also, perhaps you could rename this method to getPatientsWithEncountersBetweenDates.

Yes the intention is to get all patients lemme remove the encounter table query